redesign/generators.py

```diff
@@ -1,44 +1,55 @@
 """Contains generators for tabular, graph, and unstructured data profiles."""
 
+from synthetic_data.synthetic_data import make_data_from_report
+
 from .base_generator import BaseGenerator
 
 
 class TabularGenerator(BaseGenerator):
     """Class for generating synthetic tabular data."""
 
-    def __init__(self, profile, options):
+    def __init__(self, profile, seed=None, noise_level: float = 0.0):
         """Initialize tabular generator object."""
-        super().__init__(profile, options)
+        super().__init__(profile, seed)
+        self.noise_level = noise_level
 
-    def synthesize(self):
+    def synthesize(self, num_samples: int, seed=None, noise_level: float = None):
         """
         Generate synthetic tabular data.
         """
-        if not self.options:
-            return "Synthesized tabular data!"
+        if seed is None:
+            seed = self.seed
```

Comment:
why is `seed` still in here?

Reply:
so the user can override if they want to set it at synth

Reply:
i see